ports/templatetags/format_names.py

```diff
@@ -0,0 +1,23 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter('os_version')
+def trim_builder_name(builder_name):
+    first_trim = builder_name.replace('_x86_64', '')
+    return first_trim.replace('_legacy', '')
```

Comment:
can simply be `return builder_name.replace('_x86_64', '').replace('_legacy', '')`